winml/test/common/SqueezeNetValidator.cpp

```diff
@@ -1,14 +1,13 @@
 #include "SqueezeNetValidator.h"
```

Comment:
SqueezenetValidator doesn't send up telemetry. Why delete profiling code?

Reply:
We are not using the profiling data anywhere else, but print them out at the bottom of this file. So it would be okay to delete them all since we are not collecting perfomance data here.

---
In reply to: [355712048](https://github.com/microsoft/onnxruntime/pull/2577#discussion_r355712048) [](ancestors = 355712048)